### ISSUE TITLE
v2.8.1 missing top left 3 bar ☰ menu button

### ISSUE THREAD

**Issue by patkkleung** [rancher/dashboard#10332]
After upgraded to v2.8.1, I can no longer see the top left ☰ menu button.  Please advise.

**Comment by richard-cox** [comment]
This seems more like a possible bug. Would you be able to update the description with the following template?

Was the previous version before 2.8.0?

```
<!--------- For bugs and general issues --------->
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**
<!--A clear and concise description of what the bug is.-->

**To Reproduce**
<!--Steps to reproduce the behavior-->

**Result**

**Expected Result**
<!--A clear and concise description of what you expected to happen.-->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem and/or errors in the browser's dev console -->

**Additional context**
<!--Add any other context about the problem here. -->

```

**Comment by patkkleung** [comment]
I upgrade from 2.7.x to 2.8.0 then to 2.8.1.Please review my info below.  Thanks.PatrickOn Jan 29, 2024, at 12:43 AM, Richard Cox ***@***.***> wrote:﻿
This seems more like a possible bug. Would you be able to update the description with the following template?
Was the previous version before 2.8.0?

**Setup**
- Rancher version:
v2.8.1- Rancher UI Extensions:
no extension- Browser type & version:
Chrome v121.0.6167.85Firefox v115.7.0esr
**Describe the bug**


Missing top left 3 bar menu icon/button.  Cannot click and access Rancher configuration page and other menu items.**To Reproduce**


Login and 3 bar never show up.**Result**

no 3 bar menu at top left**Expected Result**


see 3 bar menu**Screenshots**


**Additional context**

**Comment by richard-cox** [comment]
Could you see the menu when you upgraded to 2.8.0, and it's only after upgrading to 2.8.1 you don't see it?

Can you provide some screenshots, and are there any errors in the browser's console?

Do you have any downstream clusters?

**Comment by patkkleung** [comment]
No 3 bar menu when upgraded to v2.8.0 either.  So, upgraded to v2.8.1 and
hope to see it in that version.  The error in console is this:

client.js:793 TypeError: Cannot read properties of undefined (reading
'fontSize')
    at a.globalBannerSettings (TopLevelMenu.vue:79:1)
    at e.get (vue.runtime.esm.js:3446:33)
    at e.evaluate (vue.runtime.esm.js:3547:27)
    at a.globalBannerSettings (vue.runtime.esm.js:5537:25)
    at a.sideMenuStyle (TopLevelMenu.vue:63:1)
    at e.get (vue.runtime.esm.js:3446:33)
    at e.evaluate (vue.runtime.esm.js:3547:27)
    at a.sideMenuStyle (vue.runtime.esm.js:5537:25)
    at a.Bt (TopLevelMenu.vue:1:344)
    at Tn.e._render (vue.runtime.esm.js:2684:28)

Please see the screenshot in my last email.

Patrick

**Comment by patkkleung** [comment]
Does the image below show up?PatrickOn Jan 30, 2024, at 9:46 AM, Richard Cox ***@***.***> wrote:﻿
Unfortunately there's no image.
This looks like it might be the same as #10140 though.
You might be able to work around this by going to <rancher url>/v3/settings/ui-banners and Delete the value there. Going back to rancher should then work (after refreshing the page). This does mean though you would need to apply any custom banners again.

**Comment by patkkleung** [comment]
Happy to report that after resetting the top banner and reload the page, the top left 3 bar menu reappears.  You may close the issue.  Thank you.PatrickOn Jan 30, 2024, at 1:00 PM, Patrick Leung ***@***.***> wrote:﻿Does the image below show up?<image001.jpg>PatrickOn Jan 30, 2024, at 9:46 AM, Richard Cox ***@***.***> wrote:﻿
Unfortunately there's no image.
This looks like it might be the same as #10140 though.
You might be able to work around this by going to <rancher url>/v3/settings/ui-banners and Delete the value there. Going back to rancher should then work (after refreshing the page). This does mean though you would need to apply any custom banners again.

**Comment by richard-cox** [comment]
Closing as requested, covered by https://github.com/rancher/dashboard/issues/10140. The fix is targetted for 2.9.0 and under consideration for backporting to 2.8.x
